entity-service-client/src/main/java/org/hypertrace/entity/data/service/client/EdsCacheClient.java

```diff
@@ -17,26 +17,35 @@
 import org.hypertrace.entity.data.service.v1.EntityRelationship;
 import org.hypertrace.entity.data.service.v1.EntityRelationships;
 import org.hypertrace.entity.data.service.v1.Query;
+import org.hypertrace.entity.service.client.config.EntityServiceClientCacheConfig;
 import org.hypertrace.entity.service.client.config.EntityServiceClientConfig;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class EdsCacheClient implements EdsClient {
```

Comment:
We should add unit tests to this class to unit test that the caching is effective.

Reply:
Added basic test.